### PR TITLE
JAMES-2343 buffering DeferredFileOutputStream [CONTRIB]

### PR DESCRIPTION
**Pull request by chibenwa**
CI for https://github.com/apache/james-project/pull/105

### CODE REVIEW

File: server/container/core/src/main/java/org/apache/james/server/core/MimeMessageInputStreamSource.java [hunk 1]
```diff
@@ -78,7 +79,9 @@ public MimeMessageInputStreamSource(String key, InputStream in) throws Messaging
         // Create a temp file and channel the input stream into it
         try {
             out = new DeferredFileOutputStream(THRESHOLD, "mimemessage-" + key, ".m64", TMPDIR);
-            IOUtils.copy(in, out);
+            BufferedOutputStream buffered = new BufferedOutputStream(out, THRESHOLD);
+            IOUtils.copy(in, buffered);
+            buffered.flush();
```
Comment (rouazana) [inline server/container/core/src/main/java/org/apache/james/server/core/MimeMessageInputStreamSource.java]: the buffered does not need to be closed?

Comment (chibenwa) [reply server/container/core/src/main/java/org/apache/james/server/core/MimeMessageInputStreamSource.java]: Could you make that remark directly on the contributor PR?

Comment (rouazana) [reply server/container/core/src/main/java/org/apache/james/server/core/MimeMessageInputStreamSource.java]: of course, I just wanted to be sure I was not missing something

**Comment by chibenwa** [comment]
This PR is not up to date with the contribution.

**Pull request closed without merging**